src/patternfly/components/Table/examples/Table.md

```
@@ -135,7 +135,8 @@ import './Table.css'
 
 ## Responsive table behavior
 ### Responsive layout modifier usage
-These classes can be used to ensure that the table changes between the tabular and grid-based layout at an appropriate screen width.
+These classes can be used to ensure that the table changes between the tabular and grid-based layout at an appropriate screen width. 
```

markdownlint-cli2 (0.20.0)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)